app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -25,13 +29,40 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicPlayFragmentTest {
 
+  // TODO(#137): Add following test-cases once story-progress function is implemented and expandable list is introduced.
```

Comment:
Also: test that the list appears correctly with multiple chapters (your current test only handles the single chapter case).

Reply:
Done.